Lesson9/anna/index.html

```diff
@@ -6,6 +6,395 @@
+              
+            <svg width="18px" height="18px" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-share"><path d="M4 12v8a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2v-8"></path><polyline points="16 6 12 2 8 6"></polyline></line> <button class="action-icon"></button></svg>
+         
+      </div>   <!svg leri gri yapmaya çalıştım (color="var(--color-secondary)" ama çalışmadı siyah kaldı. class verererek de denedim, html de svg nin içindeki styles kısmından da yaptım ama siyah kaldı. Hover yapıldığında da renk değişimi olmuyor->
+ </div>
+			</section>
+<section class="hero-photo-part">
+  <figure class="hero-photo-parttwo">
+  <img class="hero-content" src="https://miro.medium.com/v2/resize:fit:1100/format:webp/1*aVzJTznRRfP1lM7AXe9yLw.jpeg" alt="Photo by on Unsplash"   </figure>
+    <figcaption class="fig-text">Photo by <a class="fig-text-link" href=""> Kevin Ku</a> on <a class="fig-text-link" href="">Unsplash</a></figcaption>
+	</section>
+  <article class="parag">
+    <p class="scnd">As a web developer, there is a high probability that you have encountered the two enemies of this article:<strong> images </strong> and <strong> deadlines.</strong> Sometimes, for some reasons, your images won’t fit the layout and you don’t want to wrap your head around this for hours.</p>
+    <p class="scnd">
+      This situation has happened to me many times and I have learned from my mistakes. No more black magic hacks — here are my five favorites techniques to handle image resizing.
+    </p>
+    <h1  class="space">the “OMG I NEED THIS ASAP” way</h1>
```

Comment:
Same about this class, I'd use two separate classes to add top and bottom margin to the element. Also, I think this is not the most important heading on the page so should be something other than h1 (e.g. h2 or h3).